client/src/components/molecules/AdminSettings/index.tsx

```diff
@@ -25,27 +43,32 @@ const AdminSettings = ({ isOpen, setIsOpen }: { isOpen: boolean, setIsOpen: () =
       >
         {menu}
       </button>
-    )
+    );
   });
 
-  const [passwordData, setPasswordData] = useState<{
-    currentPassword: string
-    newPassword: string
-    newConfirmedPassword: string
-  }>({
+  const [passwordData, setPasswordData] = useState<Password>({
     currentPassword: "",
     newPassword: "",
     newConfirmedPassword: ""
   });
-  const onPasswordChange = (e: { target: { value: string, name: string } }): void => {
+  const onPasswordChange = (e: { target: { value: string, name: string; }; }): void => {
     const value = e.target.value;
```

Comment:
Instead of destructing the `e` variable, you can provide its actual type. I think that's much better.